app/views/users/show.html.erb

```diff
@@ -1,6 +1,17 @@
-+<h1>Hello User!</h1>
- +<br />
- +<p>Username: <%= @users.name %></p>
- +<p>Email: <%= @users.email %></p>
- +<p>Member since: <%= (@users.created_at).to_formatted_s(:long_ordinal) %></p>
- +<p>Last login: <%= time_ago_in_words(@users.last_sign_in_at) %></p> 
+<h1>Hello User!</h1>
+ <br />
+ <p>Username: <%= @user.name %></p>
+ <p>Email: <%= @user.email %></p>
+ <p>Member since: <%= (@user.created_at).to_formatted_s(:long_ordinal) %></p>
+ <p>Last login: <%= time_ago_in_words(@user.last_sign_in_at) %></p> 
+ <h1>
+    To do list:
+ </h1>
+  <table class="table table-striped">
+     <tbody>
+       <br/>
+         <%= render  @item %>
```

Comment:
Alternatively you could loop here through every `item` in `@items`, rendering `item` for each iteration and passing a [local variable](http://guides.rubyonrails.org/layouts_and_rendering.html#passing-local-variables) to the partial.  Both will accomplish the same goal.